internal/quickstart/toggle_flag.go

```diff
@@ -82,6 +85,9 @@ func (m toggleFlagModel) View() string {
 
 	if m.flagWasEnabled {
 		furtherInstructions = fmt.Sprintf("\n\nCheck your %s to see the change!", logTypeMap[m.sdkKind])
+		if m.hasToggleConflict {
+			furtherInstructions = fmt.Sprintf("\n\nThe flag is already toggled %v.", toggle)
+		}
```

Comment:
Showing this message when flag is already toggled to either "on" or "off" on the server